src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -49,15 +52,15 @@ public AirplaneDaoImpl(DataSource dataSource) {
                 return Optional.empty();
             }
         } catch (SQLException e) {
-            throw new DaoOperationException("Error saving product: " + airplane, e);
+            throw new DaoOperationException("Error saving airplane: " + airplane, e);
         }
     }
 
     @Override
     public List<Airplane> findAll() {
         List<Airplane> airplaneArray = new ArrayList<>();
```

Comment:
1. array and list are different data structures, it's like Cat and Dog. We can't name variables like this:
Cat dog = new Cat();
Map<String, String> set = new HashMap<String, String>; (BTW, in term of math map, list, array are sets, but we should think in terms in Java)
int[] list = {1, 2, 3}
and so on.

Also if it's temp variable, it's overhead to have such detailed name.
We are not using it outside of method, so:
-List<Airplane> airplanes
-List<Airplane> result
- List<Airplane> list
are good.

2. Also this temp variable. Variable should be declared in places they needed. We don't need this list BEFORE try.
3. so move it before WHILE (rs.next)
4. Remove RETURN at the end of method, if it should be some error, it would never returns empty list.
5. Make 1 return in this method, after While (rs.next()) {}